llo/plugin_codecs.proto

```diff
@@ -0,0 +1,55 @@
+syntax="proto3";
+
+package v1;
+option go_package = ".;llo";
+
+// WARNING
+// Use extreme caution making changes in this file
+// All changes MUST be backwards compatible
+// If the format changes in a backwards incompatible way, active DONs can
+// become stuck permanently
+message LLOObservationProto {
+	bytes attestedPredecessorRetirement = 1;
+	bool shouldRetire = 2;
+	int64 unixTimestampNanoseconds = 3;
+	repeated uint32 removeChannelIDs = 4;
+	map<uint32, LLOChannelDefinitionProto> updateChannelDefinitions = 5;
+	map<uint32, bytes> streamValues = 6;
```

Comment:
Would it be better to use a repeated message instead of a map here? A map limits you to the provided key and value types only, but a repeated message gives you the option of adding more fields later on. e.g. you already have `LLOStreamIDAndValue`.

Reply:
Nah not necessary because Observation doesn't need to be serialized deterministically